src/Donations/Components/PaymentsForm.tsx

```diff
@@ -93,6 +93,10 @@ function PaymentsForm({}: Props): ReactElement {
     method: string,
     providerObject?: any
   ) => {
+    if (!paymentSetup) {
+      console.log("Missing payment options");
+      return;
+    }
```

Comment:
is this log is necessary ?

Reply:
Yes, in the unlikely case that `paymentSetup` is not initialized, we'd be able to diagnose the problem quickly. At the same time, this isn't a message that needs to be shown to the user.

This scenario is unlikely to occur, so this log will be probably never be shown.